website/content/docs/configuration/events/common.mdx

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+page_title: Controller - Events - File Sink - Configuration
+description: |-
+  The common sink parameters for all sink types.
+---
+
+# Common Sink Parameters
+
+These parameters are shared across all sink types:
+
+- `name` - Specify a name for the sink.
+
+- `description` - Specify a description for the sink.
+
+- `event_types` - Specifies a list of event types that will be sent to the sink.
+  Can be `*`, `error`, `system`, `observation` or `audit` 
```

Comment:
May want to specify that audit does not currently produce any events so people don't wonder why they're seeing nothing. :-D

Reply:
very good point!